Corona-Warn-App/src/main/java/de/rki/coronawarnapp/srs/core/server/SrsSubmissionServer.kt

```diff
@@ -23,6 +24,7 @@ import javax.inject.Inject
 class SrsSubmissionServer @Inject constructor(
     srsSubmissionApi: Lazy<SrsSubmissionApi>,
     private val paddingTool: PaddingTool,
+    private val srsSubmissionSettings: SrsSubmissionSettings,
```

Comment:
I believe you can do it in `SrsSubmissionRepository` where `srsSubmissionSettings` is already available 
<img width="766" alt="Screenshot 2022-12-15 at 12 58 14" src="https://user-images.githubusercontent.com/25054729/207853709-93caaa1d-cfdf-4ffc-86fd-50958953890f.png">

Reply:
Ok.